install/production-filesystem/services/start-nginx.sh

```
@@ -0,0 +1,60 @@
+# Execute nginx with overrides
+# echo the full nginx command then run it
+echo "nginx	-p \"${RUN_DIR}/\" -c \"${SYSTEM_NGINX_CONFIG_FILE}\" -g \"error_log ${NETALERTX_LOG}/nginx-error.log; pid ${RUN_DIR}/nginx.pid; daemon off;\" &"
+nginx \
+	-p "${RUN_DIR}/" \
+	-c "${SYSTEM_NGINX_CONFIG_FILE}" \
+	-g "error_log ${NETALERTX_LOG}/nginx-error.log; pid ${RUN_DIR}/nginx.pid; daemon off;" &
+nginx_pid=$!
+
+wait "${nginx_pid}"
+echo -ne " done"
+exit $?
```

Shellcheck (0.11.0)

[warning] This $? refers to echo/printf, not a previous command. Assign to variable to avoid it being overwritten.

(SC2320)